src/metabase/driver/sql_jdbc/sync/describe_table.clj

```diff
@@ -388,37 +388,45 @@
   Respects *nested-field-column-max-row-length*."
   [v path]
   (let [p (json-parser v)]
-    (loop [path (or path [])
-           key  nil
-           res  (transient {})]
-      (let [token     (.nextToken p)
-            next-path (cond-> path key (conj key))]
+    (loop [path      (or path [])
+           keyy      nil
```

Comment:
ehm, is this intentional?

Reply:
yea, I don't like the fact that it shadows function `key` :D.  we do this sometimes we keys like `type` and we use `ttype` instead.

maybe `kkey` is better?

Reply:
ah! how do you like `field`? 😁